DrunkenDragon

```diff
@@ -0,0 +1,15 @@
+[
+    {
+        "project": "Pixel Tiles",
+        "tags": [
+            "PixelTile",
+            "Tavern",
+            "Inn",
```

Comment:
Multiple tags are a little too vague, please read the new entry in the readme.
Must be uniquely identifiable to your project nft names

Reply:
Ok I changed them :)